### PR TITLE
cscope version bump to 15.9

### PR DESCRIPTION
**Pull request by cineto**

### CODE REVIEW

File: dev-util/cscope/cscope-15.9.ebuild [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	emacs? ( virtual/emacs )"
+DEPEND="${RDEPEND}
+	sys-devel/flex
+	virtual/pkgconfig
+	virtual/yacc"
+
+SITEFILE="50${PN}-gentoo.el"
+
+src_prepare() {
+	eapply_user
```
Comment (hanetzer) [inline dev-util/cscope/cscope-15.9.ebuild]: `s/eapply_user/default/`. `default` eapplies the contents of `PATCHES=()` if there is one
(and there may be one later, so may as well save on future changes) and calls
`eapply_user` as well.

**Comment by hanetzer** [comment]
Squash these two commits and apply a proper commit message (which you would get
if you were using repoman [you should]) in the format `dev-util/cscope: your message here`
(`dev-util/cscope: version bump` should suffice). Also add a signed-off-by line.

File: dev-util/cscope/cscope-15.9.ebuild [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+pkg_postinst() {
+	use emacs && elisp-site-regen
+}
+
+pkg_postrm() {
+	use emacs && elisp-site-regen
+}
+
```
Comment (hanetzer) [inline dev-util/cscope/cscope-15.9.ebuild]: Remove this last line.

File: dev-util/cscope/cscope-15.9.ebuild (hunk 1 above)
Comment (ulm) [reply dev-util/cscope/cscope-15.9.ebuild]: This is entirely a question of the maintainer's style. cscope-15.8a-r2.ebuild has explicit eapply and eapply_user calls and I think that is perfectly fine.

However, while the Darwin patch has been taken care of upstream, cscope-15.7a-ocs-sysdir.patch still applies and shouldn't have been dropped.

**Pull request closed without merging**